plugins/tech-stack/src/components/ToolsComponent/ToolsComponent.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// /*
+
+import React, { FC } from 'react';
+import { Grid } from '@material-ui/core';
+import {
+  Header,
+  Page,
+  pageTheme,
+  Content,
+  ContentHeader,
+  HeaderLabel,
+  SupportButton,
+} from '@backstage/core';
+
+import ToolsCardComponent from '../ToolsCardComponent';
+
+import { ToolsListConfig } from '../../config/ToolsListConfig';
+
+import { toolsListConfigFill } from '../../utils/ToolsComponentUtils';
+
+const ToolsListConfigFilled = toolsListConfigFill(ToolsListConfig);
+
+const ToolsComponent: FC<{}> = () => {
+  return (
+    <Page theme={pageTheme.tool}>
+      <Header title="Our Tech Stacks!" subtitle="">
+        <HeaderLabel label="Owner" value="easylo" />
```

Comment:
Recommend removing this. Infers anyone who installs the plugin will see that their code is owned by you.  In general cases, I think these header labels are particularly valuable on an individual catalog entity if it's a component or something you want to show is owned by a group (your api-team) or some other team at your own company's installation? But showing your name across my firm wouldn't make sense 😁